docs/source/containerization/kubernetes.mdx

```diff
@@ -1,264 +1,270 @@
+
+```yaml title="my_values.yaml"
+extraContainers:
+  - name: <coprocessor-deployed-name> # name of deployed container
+    image: <coprocessor-app-image> # name of application image
+    ports:
+      - containerPort: <coprocessor-container-port> # must match port of router.configuration.coprocessor.url
+    env: [] # array of environment variables
 ```
 
-## The health endpoint
+* Deploy the router with the additional YAML configuration file. For example, starting with the `helm install` command from the basic deployment step, append `--values my_values.yaml`:
+
+  ```bash
+  helm install --namespace <router-namespace> --set managedFederation.apiKey="<graph-api-key>" --set managedFederation.graphRef="<graph-ref>"  oci://ghcr.io/apollographql/helm-charts/router --version <router-version> --values router/values.yaml --values my_values.yaml
+  ```
+
+## Separate configurations per environment
 
-The router supports a health endpoint. You can see from the examples above how it can be used in a kubernetes deployment.
+To support your different deployment configurations for different environments (development, staging, production, etc.), Apollo recommends separating your configuration values into separate files:
 
-If you had a router running on port 8088 on your localhost, you could exercise the health endpoint as follows:
+- A **common** file, which contains values that apply across all environments.
+- A unique **environment** file per environment, which includes and overrides the values from the common chart while adding new environment-specific values.
+
+In a `helm install` command, each `--values <values-file>` option is applied in the order within the command. Therefore, a common file must be set before an environment file in the `helm install` command, so that the environment file's values are applied last and override the commond file's values as intended. 
+
+For example, deploy with a `common_values.yaml` file applied first and then a `prod_values.yaml` file:
 
 ```bash
-curl "http://localhost:8088/health"
-{"status":"UP"}
+helm install --namespace <router-namespace> --set managedFederation.apiKey="<graph-api-key>" --set managedFederation.graphRef="<graph-ref>"  oci://ghcr.io/apollographql/helm-charts/router --version <router-version> --values router/values.yaml  --values common_values.yaml --values prod_values.yaml
 ```
 
-If you had a router running on your localhost, with default health-check configuration, you could exercise the health endpoint as follows:
+## Configure for migration from gateway
 
-curl "http://localhost:8088/health"
+When [migrating from `@apollo/gateway` to the Apollo Router](../migrating-from-gateway), consider the following tips to maximize the compatibility of your router deployment.
 
-## Using `istio` with the router
+* The gateway's maximum supported request size is 20MB, while the router's default maximum request size is less. For compatibility, set the deployed router's maximium request size to 20MB:
```

Comment:
I'm not sure about this. I think it's just that Apollo's gateway was configured to support 20MB. The router by default has a max of 2Mb, so we increased it to 20MB.

This is more general wording that will work for all users:

By default the router has a maximum supported request size of 2MB. If your gateway accepts requests larger than 2MB, you can use the following configuration to ensure that the router is compatible with your gateway deployment.

<yaml extract>